lib/trento_web/controllers/v1/users_controller.ex

```diff
@@ -179,4 +180,10 @@ defmodule TrentoWeb.V1.UsersController do
   defp attach_user_version_as_etag_header(conn, %User{lock_version: lock_version}) do
     put_resp_header(conn, "etag", Integer.to_string(lock_version))
   end
+
+  # when oidc is enabled, we only allow abilities as parameter
+  defp clean_params_for_oidc_integration(attrs, true), do: Map.take(attrs, [:abilities])
```

Comment:
We would need to take `:locked_at` as well, right?
Otherwise we cannot lock the user internally in trento

Reply:
Yeah we should whitelist the enabled field, nice catch!

Reply:
oh yes, `enabled` field, you are right